micall/utils/fetch_sequences.py

```diff
@@ -95,6 +97,27 @@ def check_hiv_seeds(project_config, unchecked_ref_names: set):
     return error_count
 
 
+def check_hivgha_seeds(project_config, unchecked_ref_names: set):
+    print("HIVGHA project uses the same seeds as HIV, plus three chimeric seeds.")
```

Comment:
It's only three chimeric seeds, I think.
I got a little confused because `ref_names` is first defined in line 102 and then over-written in 110 and 111, but I think it makes sense to me now (you want to make sure that the extra sequences are the exact same ones as the hard-coded ones, right?). It would be a little more flexible for future updates, like more HIVGHA seeds, if you move lines 105 to 108 after the second assignment of `ref_names` and leave out the initial assignment, I think.

Reply:
Good catch! We started with three chimeric refs, and I forgot to update the comment.
You're also right about `ref_names`, I hadn't noticed the name collision. It won't cause bugs, but it's confusing, so I renamed it.